src/Microsoft.ML.LightGBM/LightGbmRegressionTrainer.cs

```diff
@@ -108,10 +108,10 @@ protected override void CheckDataValid(IChannel ch, RoleMappedData data)
 
         protected override void CheckAndUpdateParametersBeforeTraining(IChannel ch, RoleMappedData data, float[] labels, int[] groups)
         {
-            Options["objective"] = "regression";
+            Options["objective"] = "regression".ToString(CultureInfo.InvariantCulture);
```

Comment:
>"regression".ToString(CultureInfo.InvariantCulture); [](start = 35, length = 52)

I have so many questions...

Reply:
99 questions but ... aint one? 

---
In reply to: [199290138](https://github.com/dotnet/machinelearning/pull/454#discussion_r199290138) [](ancestors = 199290138)